src/solvers/dgsem_p4est/dg_2d.jl

```diff
@@ -320,13 +412,21 @@ end
   # to obtain the flux of the large element.
   u_buffer .*= -2
 
-  # Copy interpolated flux values from buffer to large element face in the correct orientation
-  for i in eachnode(dg)
-    for v in eachvariable(equations)
-      # Use Tuple `node_indices` and `evaluate_index_surface` to copy flux
-      # to surface flux storage in the correct orientation
-      surface_index = evaluate_index_surface(large_indices, size_, 1, i)
-      surface_flux_values[v, surface_index, large_direction, large_element] = u_buffer[v, i]
+  # Copy interpolated flux values from buffer to large element face in the
+  # correct orientation.
+  # Note that the index of the small sides will always run forward but
+  # the index of the large side might need to run backwards for flipped sides.
+  if :i_backwards in large_indices
```

Comment:
Is it really `in large_indices`, i.e., does the position in `large_indices` not matter?

Reply:
In 2D, `large_indices` will always have one entry that is either `:one` or `:end`. The other entry is always either `:i` or `:i_backwards`. The latter one decides what we need to do here.

Reply:
:+1: